README.org

```diff
@@ -33,28 +33,48 @@ You can try out individual scripts by starting running Mumbler directly.
   ${MUMBLER_REPO}/lang/build/install/mumbler/bin/mumbler ${MUMBLER_REPO}/test/<a-random-script>.mumbler
 #+end_src
 
+You can disable tail call optimization.
+
+#+begin_src shell-script
+  ${MUMBLER_REPO}/lang/build/install/mumbler/bin/mumbler --tco=false ${MUMBLER_REPO}/test/<a-random-script>.mumbler
+#+end_src
+
+You can also run individual scripts using gradle.
+
+#+begin_src shell-script
+    cd ${MUMBLER_REPO}/lang
+    ./gradlew run -PappArgs="['/home/jchaloup/development/mumbler/benchmark/fibonacci/fibonacci.mumbler']"
+    ./gradlew run -PappArgs="['/home/jchaloup/development/mumbler/benchmark/fibonacci/fibonacci.mumbler', '--tco=false']"
+#+end_src
+
 ** Run benchmarks
 [[sec:build][Build]] Truffle and Mumbler and [[sec:env-var][set environment variables]].
 
 #+begin_src shell-script
     ${MUMBLER_REPO}/benchmark/measure <benchmark-name>
 #+end_src
 
-~benchmark-name~ is the name of one of the directories under ~benchmark~.
+~benchmark-name~ is the name of the files in one of the directories under ~benchmark~, without the extension, as in the following example.
+
+#+begin_src shell-script
+    ${MUMBLER_REPO}/benchmark/measure ${MUMBLER_REPO}/benchmark/fibonacci/fibonacci
+#+end_src
```

Comment:
Using a code block here implies that this is a valid example (unless otherwise specified), but this code wouldn't work. Instead, probably change to something like

```suggestion
~benchmark-name~ is the name of the files in one of the directories under ~benchmark~, without the extension. For example, ~fibonacci~ in

#+begin_src shell_script
    ${MUMBLER_REPO}/benchmark/measure fibonacci
#+end_src

will look for scripts named ~fibonacci~ under the ~${MUMBLER_REPO}/benchmark/fibonacci~ directory.
```

Reply:
I actually use: `./benchmark/measure ./benchmark/fibonacci/fibonacci` to run the fibonacci benchmark from the mumbler repo directory. Using just `./benchmark/measure fibonacci` didn't work for me even before I did changes in the measure script. So the above example is valid one. Are you OK with keeping the instructions as they are?

Reply:
Sure. If running the script requires the path, then keeping as-is sounds good to me